src/client/components/componentWithSliders/savePressetModalBlock/backgroundUnderModelWindow/backgroundUnderModelWindow.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './backgroundUnderModelWindow.css';
+
+const BackgroundUnderModelWindow = (props) => {
+  const { backgroundClick } = props;
```

Comment:
You can destruct this in function parameters

Reply:
sorry, what do you mean? this is undefined in arrow func.?